plugins/woocommerce/package.json

```diff
@@ -1,7 +1,7 @@
 {
 	"name": "woocommerce",
 	"title": "WooCommerce",
-	"version": "6.7.0",
+	"version": "7.0.0",
```

Comment:
This change can be made by the version bump tool after code freeze, but I suppose its not hurting anything here